src/logic/codePreprocessor/stringifiedAlgorithm.ts

```diff
@@ -0,0 +1,136 @@
+import { Opaque } from "../../utlis";
```

Comment:
Dynamiczne mapowanie algorytmu nie było dobrą decyzją z kilku powodów:
1. Typy  z Typescript'a 
2.  Minifikacji kodu (przez co algorytm wyglądał brzydko w zbuildowanej wersji aplikacji)
3. To że TS kompiluje się do JS, przez co pojawiały się dziwne nazwy zmiennych

W związku z tym zrezygnowałem z tego i użyłem prostszego podejścia:
algorytm jest zapisany jako string, ale w miejsca funkcji które my które my chemy podmienić wstawia odpowiedni kontent.(czyli dynamicznie utworzone TODO)